frontend/src/credit_transfers/components/CreditTransferVisualRepresentation.js

```diff
@@ -50,6 +50,30 @@ class CreditTransferVisualRepresentation extends Component {
     );
   }
 
+  _creditTransferIcon () {
+    if (Number(this.props.numberOfCredits) === 0) {
+      return { icon: 'minus', className: '' };
+    }
+
+    if (this.props.totalValue === 0.0) {
+      if (this.props.tradeType.id === CREDIT_TRANSFER_TYPES.buy.id) {
+        return { icon: 'arrow-left', className: '' };
+      }
+      if (this.props.tradeType.id === CREDIT_TRANSFER_TYPES.sell.id) {
+        return { icon: 'arrow-left', className: 'fa-flip-horizontal' };
+      }
+    }
+
+    if (this.props.tradeType.id === CREDIT_TRANSFER_TYPES.buy.id) {
```

Comment:
The result for this one and the next one seems to be the same. Can the conditions be merged together?

Reply:
@amichard I've pushed a commit simplifying the conditional.